index.html

```diff
@@ -8,12 +8,57 @@
 
   <title>Collaboration and Contribution</title>
   <link rel="stylesheet" href="src/css/style.css">
-
-  <script defer type="module" src="dist\out-tsc\index.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
+  <script>
+    $(document).ready(function(){
+      $('#icon').click(function(){
+        $('ul').toggleClass('show');
+        $('#icon i').toggleClass('fa-bars fa-times');
+      })
+    })

```

Comment:
use vanilla type script instead of jquery. and add the code to the index.ts file. for separation of concerns